### PR TITLE
Add complete call sequence

### PR DESCRIPTION
**Pull request by Gnito**

### CODE REVIEW

File: src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md [hunk 1]
```diff
@@ -142,3 +142,162 @@ book a listing, there are couple of changes needed:
...
+```js
+=> sdk.messages.send({ transactionId: orderId, content: message })
+```
+
+FTW sends an initial message to transaction if customer has added a
+message.
+
+#### Step 5.
+
+As a final ste, we need to save the payment method, if customer has
```
Comment (bladealslayer) [inline src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md]:
```suggestion
As a final step, we need to save the payment method, if customer has
```

File: src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md [hunk 2]
```diff
@@ -142,3 +142,162 @@ book a listing, there are couple of changes needed:
...
+selected the "Save for later use" checkbox. So, this is relevant if user
+has selected onetime payment - instead of making a charge from the
+previously saved credit card.
+
+On FTW, we call
+[avePaymentMethod function](https://github.com/sharetribe/flex-template-web/blob/master/src/ducks/paymentMethods.duck.js#L200)
```
Comment (bladealslayer) [inline src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md]:
```suggestion
[savePaymentMethod function](https://github.com/sharetribe/flex-template-web/blob/master/src/ducks/paymentMethods.duck.js#L200)
```

File: src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md [hunk 3]
```diff
@@ -142,3 +142,162 @@ book a listing, there are couple of changes needed:
...
+that creates stripe customer and adds updates default payment method.
+
+There are 3 different scenarios, which require different calls to Flex
+API:
+
+**1. No StripeCustomer entity connected to Flex API:**
+
+```js
+=> sdk.stripeCustomer.create({ stripePaymentMethodId }, { expand: true, include: ['defaultPaymentMethod'] })
```
Comment (bladealslayer) [inline src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md]: Is `{ stripePaymentMethodId }` the same as `{ stripePaymentMethodId: stripePaymentMethodId }`?

File: src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md [hunk 4]
```diff
@@ -142,3 +142,162 @@ book a listing, there are couple of changes needed:
...
+```js
+=> sdk.transactions.initiate({ processAlias, transition: 'transition/request-payment', params: { listingId, bookingStart, bookingEnd, setupPaymentMethodForSaving: true }})
+```
+
+**When you use previously saved payment card**, the id of Stripe's
+payment method needs to be sent to Flex API as `paymentMethod`, when
+requesting payment.
+
+```js
+=> sdk.transactions.initiate({ processAlias, transition: 'transition/request-payment', params: { listingId, bookingStart, bookingEnd, paymentMethod: stripePaymentMethodId }})
```
Comment (bladealslayer) [inline src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md]: You could break pretty much all the examples into multiple lines for readability.

File: src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md [hunk 5]
```diff
@@ -142,3 +142,162 @@ book a listing, there are couple of changes needed:
...
+- API creates PaymentIntent against Stripe API and returns
+  stripePaymentIntentClientSecret inside transaction's protectedData.
+- Booking is created at this step - so, availability management will
+  block dates for conflicting bookings.
+- Automatic expiration happens in 15 minutes, if process is not
+  transitioned to 'transition/confirm-payment' before that.
+- After this call, created transaction is saved to session storage in
+  FTW (or existing enquiry tx is updated).
+
+**When you save card details**, a new parameter needs to be passed if
```
Comment (bladealslayer) [inline src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md]:
```suggestion
**When you intend to save card details**, a new parameter needs to be passed if
```

File: src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md [hunk 6]
```diff
@@ -142,3 +142,162 @@ book a listing, there are couple of changes needed:
...
+card details are saved at the same time: `setupPaymentMethodForSaving`
+
+```js
+=> sdk.transactions.initiate({ processAlias, transition: 'transition/request-payment', params: { listingId, bookingStart, bookingEnd, setupPaymentMethodForSaving: true }})
+```
+
+**When you use previously saved payment card**, the id of Stripe's
```
Comment (bladealslayer) [inline src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md]:
```suggestion
**When you are using previously saved payment card**, the id of Stripe's
```

File: src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md (hunk 3 above)
Comment (Gnito) [reply src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md]: Yes

File: src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md (hunk 4 above)
Comment (Gnito) [reply src/docs/background/16-how-saving-payment-card-works-in-ftw/index.md]: I didn't notice that the extra "=>" string was a problem for Prettier.

**Pull request merged**